src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -78,9 +81,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    Message message = new Message(user, text);
-    datastore.storeMessage(message);
+    //Perform sentiment analysis when the user submits a message
+    Document doc = Document.newBuilder()
+        .setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    double score = sentiment.getScore();
+    //System.out.println("Sentiment Score = " + score);
```

Comment:
deadcode

Reply:
I am using this final score obtained to generate the message in line 93 though. I don't think I fully understand what you mean bu deadcode. Can you please explain this a little more?

Reply:
Oh deadcode means that its lines of code not used. For example, this line has been commented out, so it should be removed.

Reply:
Ohh, ok thanks!